### PR TITLE
Use AYON settings - Part 1

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
## Changelog Description
Reduce AYON settings conversion from `ayon_settings.py`. More PRs will come.

## Additional info
All places that use the settings are expecting new structure of the settings. In some cases (color values) were changed types in settings to match expected value.

Some conversions are "questionable" for future, but that is not a goal of this PR.

## Testing notes:
Create all packages, ayon-core and server addon. Upload them to AYON.
- [x] Applications can be launched.
- [x] Maya settings should be applied to plugins - ExtractPlayblast, ExtractThumbnail, Validations etc. Colors of loaders are propagated.
- [x] Blender extract playblast/thumbnail.
- [x] Max PointCloud extraction.
- [ ] Deadline settings are loaded.

### CODE REVIEW

**Comment by simonebarbieri** [comment]
I'm getting this error when running the launcher:
```
!!! AYON crashed:
Traceback (most recent call last):
  File "C:\Dev\Ynput\ayon-launcher\start.py", line 827, in main_cli
    cli.main()
  File "C:\Dev\Ynput\ayon-core\client\ayon_core\cli.py", line 265, in main
    _set_global_environments()
  File "C:\Dev\Ynput\ayon-core\client\ayon_core\cli.py", line 209, in _set_global_environments
    general_env = get_general_environments()
  File "C:\Dev\Ynput\ayon-core\client\ayon_core\settings\lib.py", line 259, in get_general_environments
    return json.loads(settings["core"]["environments"])
KeyError: 'environments'
```

**Comment by iLLiCiTiT** [comment]
> I'm getting this error when running the launcher:

Fixed. You have to recreate ayon-core package and upload to server.

File: client/ayon_core/plugins/publish/extract_review.py [hunk 1]
```diff
@@ -1503,14 +1511,16 @@ def filter_output_defs(self, profile, subset_name, families):
             subset_name (str): name of subset
 
         Returns:
-            list: Containg all output definitions matching entered families.
+            dict[str, Any]: Containg all output definitions matching entered
```
Comment (BigRoy) [inline client/ayon_core/plugins/publish/extract_review.py:1514]:
```suggestion
            dict[str, Any]: Containing all output definitions matching entered
```

File: client/ayon_core/hosts/photoshop/plugins/publish/collect_review.py [hunk 2]
```diff
@@ -6,24 +6,17 @@
...
-from ayon_core.pipeline.create import get_subset_name
-
 
 class CollectReview(pyblish.api.ContextPlugin):
     """Adds review to families for instances marked to be reviewable.
     """
 
-    label = "Collect Review"
-    label = "Review"
+    label = "Collect Review Family"
```
Comment (iLLiCiTiT) [inline client/ayon_core/hosts/photoshop/plugins/publish/collect_review.py:16]:
```suggestion
    label = "Collect Review"
```

**Review by simonebarbieri — APPROVED** [review]
Blender works fine!

**Review by moonyuet — APPROVED** [review]
Looks good in Max

**Comment by BigRoy** [comment]
@simonebarbieri @moonyuet you should be able to check of the checkboxes in the "Testing Notes" if you've in particular checked that particular step, like pointcloud extraction in max and playblast/thumbnail in Blender, etc.

**Review by tokejepsen — CHANGES_REQUESTED** [review]
Submitting render from Maya to Deadline gives me this:
```
Traceback (most recent call last):
  File "C:\Users\tokejepsen\AppData\Local\Ynput\AYON\dependency_packages\ayon_2310271602_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
    runner(*args)
  File "C:\Users\tokejepsen\ayon-core\client\ayon_core\hosts\maya\plugins\publish\validate_render_image_rule.py", line 29, in process
    self.get_default_render_image_folder(instance)
  File "C:\Users\tokejepsen\ayon-core\client\ayon_core\hosts\maya\plugins\publish\validate_render_image_rule.py", line 63, in get_default_render_image_folder
    return instance.context.data.get('project_settings')\
AttributeError: 'NoneType' object has no attribute 'get'
```
Although this seems unrelated to the settings?

**Comment by iLLiCiTiT** [comment]
> Submitting render from Maya to Deadline gives me this:

Should be fixed.

**Review by tokejepsen — CHANGES_REQUESTED** [review]
Nearly got a submission:
```
Traceback (most recent call last):
  File "C:\Users\tokejepsen\AppData\Local\Ynput\AYON\dependency_packages\ayon_2310271602_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
    runner(*args)
  File "C:\Users\tokejepsen\ayon-core\client\ayon_core\modules\deadline\abstract_submit_deadline.py", line 456, in process
    self.plugin_info = self.get_plugin_info()
  File "C:\Users\tokejepsen\ayon-core\client\ayon_core\modules\deadline\plugins\publish\submit_maya_deadline.py", line 260, in get_plugin_info
    instance.context.data['project_settings']
KeyError: 'RenderSettings'
```

**Comment by tokejepsen** [comment]
Maya settings works. Also done Maya Deadline submission and works as well.

**Review by tokejepsen — APPROVED** [review]

**Pull request merged**